src/modules/company/index.ts

```diff
@@ -74,6 +75,7 @@ export class CompanyModule {
       since: '8.0',
       until: '9.0',
     });
+    // eslint-disable-next-line deprecation/deprecation
     return this.faker.helpers.arrayElement(this.suffixes());
```

Comment:
In this case it is suggested to minimize the covered deprecation function by only commenting its usage

```suggestion
    return this.faker.helpers.arrayElement(
      // eslint-disable-next-line deprecation/deprecation
      this.suffixes()
    );
```

Should look something like this, might get formatted differently by prettier